server/controllers/categoryController.js

```diff
@@ -257,34 +259,60 @@ class CategoryController {
    */
   static getAllArticlesForACategory(req, res, next) {
     const { categoryName } = req.params;
+    const { query } = req;
+    const limit = Number(query.limit) || 12;
```

Comment:
Great job! Your commit messages say a limit of 6 articles but your code is showing 12. Are my missing something?

Reply:
@waleCloud I now have 2 commit messages for the feature